conda-forge.yml

```diff
@@ -1,6 +1,6 @@
-conda_forge_output_validation: true
-upload_on_branch: master
-build_platform:
-  osx_arm64: osx_64
+build_platform: {osx_arm64: osx_64}
```

Comment:
You decided to build universal, which is in my opinion a bit against the kind of conda (eg in regards to static libraries, or rust, go, etc packages).  The same I noticed in regards of pyinstall (which is btw pretty broken for osx-arm64 as soon as you don't build it universal, as they have a bug about selecting arch arm64 only).
Anyway changes look ok and are conda-forge only anyway.

Reply:
Thanks

Reply:
We are not building universal binaries. (We had to build one for Perl, but that's it. All other packages are arm64 only)

Can you open an issue about `pyinstaller`?